### PR TITLE
postgresqlPackages.pg_net: 0.8.0 -> 0.11.0

### PR DESCRIPTION
**Pull request by samrose**
Update pg_net to 0.11.0

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by samrose** [comment]
@NixOS/nixpkgs-merge-bot merge

**Review by wolfgangwalther — CHANGES_REQUESTED** [review]
> `@NixOS/nixpkgs-merge-bot merge`

I don't think you can reasonably expect to be able to use this command, for which you need to be a maintainer already, while adding yourself as one in the same PR. Independent of whether that's already in by-name or  not.

File: pkgs/servers/sql/postgresql/ext/pg_net.nix [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
 
 stdenv.mkDerivation rec {
   pname = "pg_net";
   version = "0.11.0";
 
-  buildInputs = [ curl postgresql ];
+  buildInputs = [
+    curl
+    postgresql
+  ];
```
[the remark this answers is not in the document]
Comment (samrose) [reply pkgs/servers/sql/postgresql/ext/pg_net.nix:16]: There is a check criteria that will fail a PR if you don't apply this `nixfmt` to the PR for what it's worth.

I'll leave this as it is based on comment from @Mic92 if that works?

**Comment by samrose** [comment]
> > `@NixOS/nixpkgs-merge-bot merge`
> 
> I don't think you can reasonably expect to be able to use this command, for which you need to be a maintainer already, while adding yourself as one in the same PR. Independent of whether that's already in by-name or not.

Thanks for that info. It's been some time since I have put through PR into nixpkgs, and it wasn't clear from contributing docs what various permissions are needed for these actions. But that makes sense and thank you for the clarification.

File: pkgs/servers/sql/postgresql/ext/pg_net.nix [hunk 2]
```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
     description = "Async networking for Postgres";
     homepage    = "https://github.com/supabase/pg_net";
     changelog   = "https://github.com/supabase/pg_net/releases/tag/v${version}";
-    maintainers = with maintainers; [ thoughtpolice ];
+    maintainers = with maintainers; [ thoughtpolice samrose ];
```
[the remark this answers is not in the document]
Comment (samrose) [reply pkgs/servers/sql/postgresql/ext/pg_net.nix]: done in https://github.com/NixOS/nixpkgs/pull/353532/commits/de015f4d6b384c05f31f89abe3d17e9c9e83b0b9

**Review by Ma27 — APPROVED** [review]
Except for @wolfgangwalther's remarks the change looks good to me, thanks!

File: pkgs/servers/sql/postgresql/ext/pg_net.nix (hunk 2 above)
[the remark this answers is not in the document]
Comment (khaneliman) [reply pkgs/servers/sql/postgresql/ext/pg_net.nix]: This is still in this commit and then you were just removed in a later commit.

**Pull request closed without merging**